### PR TITLE
Feat/saved palettes

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Added saved palettes section with styling to scale.
- Added save palette button.
- Added a function that displayed saved palettes in the saved palette section.
- Added message saying no palettes yet in section.

Did this break anything?
- [x] No
- [ ]  Yes
Type of change
- [x] New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.
Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
What's next?
Adding an 'X' icon next to each saved palette with the ability to delete the specific palettes in that section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,10 +57,21 @@ <h1>COLORANDOM</h1>
...
+    <h2>Saved Palettes</h2>
+    <div class="saved__palettes">
+      <p class="saved__message">No saved palettes yet!</p>
+    </div>
+  </section>
 </body>
-</html>
\ No newline at end of file
+</html>
+<script src="./main.js"></script>
```
Comment (JamieCaudill) [inline index.html:77]: Do we want to keep the script tag here or move it elsewhere?

**Review by rachelsoae — APPROVED** [review]
Looks great. Everything is functional. Once delete functionality is done (next iteration) I would like us to spend some time on refactoring the CSS and fine-tuning spacing. 
Thanks!

File: styles.css [hunk 2]
```diff
@@ -5,40 +5,53 @@ html {
...
+  border-right: solid 5px black;
 }
 
 h1 {
-  font-size: 7rem;
-  font-weight: 900;
+  height: 10%;
+  width: 100%;
+  font-size: 7em;
+  font-weight: 800;
```
Comment (JamieCaudill) [inline styles.css:27]: I would like to do some research on how to make font size more dynamic with the size of the screen.

File: index.html (hunk 1 above)
Comment (rachelsoae) [reply index.html:77]: Oh, good call. I think it needs to be inside the closing body tag.

File: styles.css [hunk 3]
```diff
@@ -64,16 +77,27 @@ h1 {
...
+  height: 30%;
+  width: 100%;
+  font-size: 1.7em;
   font-weight: 400;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  overflow: hidden;
+
```
Comment (JamieCaudill) [inline styles.css:90]: Delete this space.

File: styles.css [hunk 4]
```diff
@@ -83,25 +107,73 @@ p {
...
-  height: 3em;
-  margin-bottom: .4em;
+  height: 2em;
 }
 
 .hidden {
   display: none;
+}
+
+/* saved section */
```
Comment (JamieCaudill) [inline styles.css:136]: Do we need all these organizational comments?

File: styles.css [hunk 5]
```diff
@@ -83,25 +107,73 @@ p {
...
+  align-items: center;
+  overflow: hidden;
+}
+
+h2 {
+  font-size: 4rem;
+  font-weight: 700;
+  overflow: hidden;
+}
+.saved__palettes {
```
Comment (JamieCaudill) [inline styles.css:153]: Space here.

**Review by JamieCaudill — APPROVED** [review]
There are a few syntactical changes we can make to clean this up, but other than that, this code looks great, and it works. Next we will move on to iteration 4 and create a function that allows the user to delete saved palettes.

File: styles.css (hunk 4 above)
Comment (rachelsoae) [reply styles.css:136]: Let's review this in our final refactor phase.

File: index.html (hunk 1 above)
Comment (fernandorobles97) [reply index.html:77]: Good idea, I've moved it up before the body and HTML closing tags and will push these changes.

File: styles.css (hunk 3 above)
Comment (fernandorobles97) [reply styles.css:90]: Got it!

File: styles.css (hunk 5 above)
Comment (fernandorobles97) [reply styles.css:153]: Done

**Review by vsriqui — APPROVED** [review]
- The displaySavedPalettes function looks very clean.
- Aesthetically, the changes are beautiful.

File: styles.css (hunk 2 above)
Comment (fernandorobles97) [reply styles.css:27]: Great idea!

File: styles.css (hunk 4 above)
Comment (fernandorobles97) [reply styles.css:136]: I don't think so. We can discuss this at our next meeting!

**Comment by rachelsoae** [comment]
All reviews completed. We have some features to look forward to in the next iteration.
Thanks for making those changes, Fernando!

**Pull request merged**